File: keyfinder/src/main/java/com/convergencelabstfx/keyfinder/keypredictor/Phrase.java
package com.convergencelabstfx.keyfinder.keypredictor;

import java.util.LinkedList;
import java.util.List;

/**
 * TODO: documentation
 */
public class Phrase {

    private List<Integer> notes = new LinkedList<>();

    public Phrase() {
    }

    public List<Integer> getNotes() {
        return notes;
    }

    public void setNotes(List<Integer> notes) {
        this.notes = notes;
    }

    public void addNote(Integer toAdd) {
        notes.add(toAdd);
    }

    public Integer removeNoteAtIx(int ix) {
        return notes.remove(ix);
    }

}

File: keyfinder/src/main/java/com/convergencelabstfx/keyfinder/keypredictor/Phrase.kt
package com.convergencelabstfx.keyfinder.keypredictor

/**
 * TODO: documentation
 */
class Phrase {

    var notes: MutableList<Int> = arrayListOf()

    fun addNote(toAdd: Int) {
        notes.add(toAdd)
    }

    fun removeNoteAtIx(ix: Int): Int {
        return notes.removeAt(ix)
    }

}